index.html

```diff
@@ -4,484 +4,528 @@
-                <form class="search-container__search-form" method="get">
-                  <img
-                    class="search-container__search-icon"
-                    src="./src/icons/search-icon.svg"
-                    alt="search"
-                  />
-                  <input
-                    class="search-input"
-                    type="text"
-                    placeholder="Job title or Keyword"
-                    name="text"
-                    value=""
-                  />
-                </form>
+      <div class="container header-wrapper">
+        <a class="logo link-black" href="#">
+          J<span class="decor_orange">O</span>BPLY
+        </a>
 
-                <!-- Location form -->
-                <img src="./src/icons/line.svg" alt="icon" />
-                <form class="search-container__location-form" method="get">
-                  <img
-                    class="search-container__location-icon"
-                    src="./src/icons/location-icon.svg"
-                    alt="location"
-                  />
-                  <input
-                    class="location-input"
-                    type="text"
-                    placeholder="Location"
-                    name="text"
-                    value=""
-                  />
-                </form>
+        <!-- Navigation menu -->
+        <nav class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="nav-menu__list-item"><a href="#home">Home</a></li>
+            <li class="nav-menu__list-item">
+              <a href="#categories">Categories</a>
+            </li>
+            <li class="nav-menu__list-item"><a href="#service">Service</a></li>
+            <li class="nav-menu__list-item"><a href="#jobs">Jobs</a></li>
+            <button class="button nav__button" href="#">Post a vacancy</button>
+          </ul>
+        </nav>
+        <nav class="menu nav-menu">
+          <!-- Navigation list items -->
+          <ul class="menu__list">
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#home">Home</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#categories">Categories</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#service">Service</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#jobs">Jobs</a>
+            </li>
+          </ul>
+        </nav>
 
-                <!-- Search button -->
-                <div class="hero-section__button">
-                  <a class="buttons buttons__search-button" href="#">
-                    Search
-                  </a>
-                </div>
-              </div>
-            </div>
+        <!-- Navigation button -->
+        <button class="button button__cta" href="#">Post a vacancy</button>
```

Comment:
```suggestion
        <button class="button button_cta" href="#">Post a vacancy</button>
```
"cta" is a modifier. Please, distinguish elements and modifiers.